plugins/mac-virtualcam/src/camera-extension/OBSCameraDeviceSource.swift

```diff
@@ -0,0 +1,292 @@
+//
+//  OBSCameraDeviceSource.swift
+//  mac-camera-extension
+//
+//  Created by Sebastian Beckmann on 30.09.22.
+//  Changed by Patrick Heyer on 16.10.22.
+//
+
+import AppKit
+import CoreMediaIO
+import Foundation
+import IOKit.audio
+import os.log
+
+let OBSCameraFrameRate: Int = 60
+
+@available(macOS 13.0, *)
```

Comment:
With the deployment target set to 13.0, are the availability checks needed?

Reply:
Could you try running it without the guard in a macOS 12 VM?

Reply:
Negative, can't log into iCloud on a VM (that's apparently normal, but means I can't build the app with entitlements) and my VM is unable to boot into recovery for some reason ("Internal Virtualization Error"). Would have to install macOS 12 genuinely.